app/utils/number.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+module.exports = str => {
+  return !isNaN(str) && !isNaN(parseInt(str))
```

Comment:
The [`parseInt(…)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) function will parse things that aren’t Arabic numerals (for example, it will allow hexadecimal).

Rather than trying to parse and integer, it’s probably easier to just use a regex:

```javascript
if (displayName.match(/.*[0-9]{4}$/)) {
  // …
}
```